### DISCUSSION TITLE
Render order sometimes not Correct

### DISCUSSION THREAD

**Discussion by Boxter5000** [ppy/osu#24852] [Q&A] [answered]
### Type

Game behaviour

### Bug description

Encounterd a Bug where the Hit Circles render bihind a Slider 

https://osu.ppy.sh/beatmapsets/1503038#osu/3079348

[Reizouko Mitara Pudding ga Nai] by bibuko
Time: 1:17

### Screenshots or videos

![Screenshot 2023-09-18 203910](https://github.com/ppy/osu/assets/25961845/afa8a55a-d570-4a1c-bcd3-413d83460fbe)

### Version

2023.815.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/12652174/database.log)
[input.log](https://github.com/ppy/osu/files/12652175/input.log)
[network.log](https://github.com/ppy/osu/files/12652176/network.log)
[performance.log](https://github.com/ppy/osu/files/12652177/performance.log)
[runtime.log](https://github.com/ppy/osu/files/12652178/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12652179/updater.log)

**Comment by bdach** [comment]
Please link the beatmap. Timestamp would be nice too.

**Comment by peppy** [comment, marked as answer]
For the record, this looks normal and is how stacking works on slider tails. Unless I'm missing some context not captured in the screenshot.

**Reply by Boxter5000** [reply]
I guess it is just now that i Try the argon Skin i noticed that nodes stack like this because the Sliders are not Transparent. I checked other maps and noticed that there it is like this as well. So it is not a bug i guess. But still tbh i find it quite distracting not beeing able to see the more or less Full note behind the slider end. If more people tink the same way then maby it is a disgustion to change the Skin a bit.
